alembic/versions/a5a4841351d7_refactor_response_model.py

```diff
@@ -20,7 +20,7 @@
 def upgrade():
     # ### commands auto generated by Alembic - please adjust! ###
     questiontype = postgresql.ENUM(
-        'check', 'text_area', 'rate', name='questiontype')
+        'check', 'text_area', 'rate', 'missing_items', name='questiontype')
```

Comment:
@mnswaleh you seem to be editing an already existing alembic file. The changes you are introducing here will be put into effect. What were you trying to achieve here?

Reply:
The missing items option was missing on the responses question_type field. The change is meant to introduce that option on the database to avoid the error that was occurring due to the missing option